### PR TITLE
User persona post added

### PR DESCRIPTION
**Pull request by jesseyuen**

### CODE REVIEW

File: app/posts/publish-teacher-training-courses/2020-08-20-user-personas.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
+---
+title: User personas as test accounts in Publish
+description: Viewing Publish from the perspective of our users
+date: 2020-08-20
+---
+
+{% from "figure/macro.njk" import appFigure with context %}
+
+When joining the Find/Publish team members had traditionally been granted administrator access to the QA Publish environment.
```
Comment (edwardhorsford) [inline app/posts/publish-teacher-training-courses/2020-08-20-user-personas.md]:
```suggestion
When joining the Find or Publish teams, team members had traditionally been granted administrator access to the QA Publish environment.
```

File: app/posts/publish-teacher-training-courses/2020-08-20-user-personas.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+* multiple accredited bodies, Suffolk and Norfolk Primary SCITT and Suffolk and Norfolk Secondary SCITT
+* Thorpe St Andrew School and Sixth Form, a school direct with courses accredited by Suffolk and Norfolk Secondary SCITT as well as courses accredited by an organisation not associated with either of Mary’s accredited bodies.
+
+### Colin
+
+Colin is DfE support and has administrator access to all organisations. 
+
+## Logging in as a specific user
+
+For troubleshooting a specific user's issues Support are able to login with the users anonymised email using the ‘Sign in as a specific user’ link in the header. You can also sign in as yourself this way.
```
Comment (edwardhorsford) [inline app/posts/publish-teacher-training-courses/2020-08-20-user-personas.md]:
```suggestion
For troubleshooting a specific user’s issues, DfE Support are able to login with the user's anonymised email using the ‘Sign in as a specific user’ link in the header. You can also sign in as yourself this way.
```

File: app/posts/publish-teacher-training-courses/2020-08-20-user-personas.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+* with approximately 2000 providers it’s often an hard to find the appropriate organisation for testing a specific scenario
+* our users often have complex relationships across multiple organisations which we can’t visualise as an admin user
+
+## Test accounts
+
+We’ve created test accounts based on existing anonymised users for both QA and review apps when testing PRs. The test accounts represent common user personas identified in the service and will help in testing new features and building more empathy with our users. 
+
+### Anne
+
+Anne is a course administrator who belongs to Groby and Partners, a school direct.
```
Comment (edwardhorsford) [inline app/posts/publish-teacher-training-courses/2020-08-20-user-personas.md]:
```suggestion
Anne is a course administrator who belongs to Groby and Partners, a school direct school.
```

Does this read weirdly to you? Should it be capitalised?

File: app/posts/publish-teacher-training-courses/2020-08-20-user-personas.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+### Mary
+
+Mary is a SCITT Business Support Coordinator who belongs to:
+
+* multiple accredited bodies, Suffolk and Norfolk Primary SCITT and Suffolk and Norfolk Secondary SCITT
+* Thorpe St Andrew School and Sixth Form, a school direct with courses accredited by Suffolk and Norfolk Secondary SCITT as well as courses accredited by an organisation not associated with either of Mary’s accredited bodies.
+
+### Colin
+
+Colin is DfE support and has administrator access to all organisations.
```
Comment (edwardhorsford) [inline app/posts/publish-teacher-training-courses/2020-08-20-user-personas.md]: Should these be alphabetical?

File: app/posts/publish-teacher-training-courses/2020-08-20-user-personas.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+
+### Colin
+
+Colin is DfE support and has administrator access to all organisations. 
+
+## Logging in as a specific user
+
+For troubleshooting a specific user's issues Support are able to login with the users anonymised email using the ‘Sign in as a specific user’ link in the header. You can also sign in as yourself this way.
+
+## There is something about Mary
```
Comment (edwardhorsford) [inline app/posts/publish-teacher-training-courses/2020-08-20-user-personas.md]:
```suggestion
## There’s something about Mary
```

File: app/posts/publish-teacher-training-courses/2020-08-20-user-personas.md [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+
+The Mary persona represents a specific and potentially problematic scenario in Publish that has [emerged in recent research](/publish-teacher-training-courses/users-with-multiple-organisation-access/#limiting-access-to-some-courses). 
+
+Mary has access to multiple organisations and is associated with two accredited bodies. 
+
+Mary also has access to a school direct (Thorpe St Andrew School and Sixth Form) with courses accredited by one of the accredited bodies (Suffolk and Norfolk Secondary SCITT) they belong to.
```
Comment (edwardhorsford) [inline app/posts/publish-teacher-training-courses/2020-08-20-user-personas.md]: A school direct what?

**Review by edwardhorsford — CHANGES_REQUESTED** [review]

File: app/posts/publish-teacher-training-courses/2020-08-20-user-personas.md (hunk 4 above)
Comment (jesseyuen) [reply app/posts/publish-teacher-training-courses/2020-08-20-user-personas.md]: yeah why not!

File: app/posts/publish-teacher-training-courses/2020-08-20-user-personas.md (hunk 6 above)
Comment (jesseyuen) [reply app/posts/publish-teacher-training-courses/2020-08-20-user-personas.md]: I decided it's not important. Deleted it!

**Review by edwardhorsford — APPROVED** [review]

**Pull request merged**